test/crash/src/test/java/io/narayana/lra/arquillian/LRACoordinatorRecoveryIT.java

```diff
@@ -128,11 +128,12 @@ public void after() {
 
         // Cleans Object Store in case there are still active transactions
         clearRecoveryLogFromObjectStore();
-        clearRecoveryLogFromFS();
 
         for (Map.Entry<String, String> entry : containerDeploymentMap.entrySet()) {
             stopContainer(entry.getKey(), entry.getValue());
         }
+        clearRecoveryLogFromFS();
```

Comment:
I understand this is clean up after the server is stopped so it's fine with me for this to move. I would be tempted to see if it's worth (in a follow up) being able to assert a fail if there is something there but this is just a thought and maybe some tests would still expect some file system records

Reply:
Good point Tom. I think we wanted this clean up to prevent a test failure from contaminating  following tests and appear (in the CI) that multiple tests have failed.

Reply:
Just to be clear I see this topic as different to https://github.com/jbosstm/lra/pull/27#discussion_r1910163943 but I will resolve this one

Reply:
Actually, I am not mandating the fail checks being added but to add readability of my review I will unresolve it but it does not need addressing IMO

Reply:
In other words, if this test has left a record in the store, I think an appropriate change should be made to fail() (but that does not block me approving this review)